components/org.wso2.carbon.business.rules.core/src/main/java/org/wso2/carbon/business/rules/core/services/TemplateManagerService.java

```diff
@@ -97,6 +98,37 @@ public void createBusinessRuleFromTemplate(BusinessRuleFromTemplate businessRule
         }
     }
 
+    public void createBusinessRuleFromScratch(BusinessRuleFromScratch businessRuleFromScratch) {
+        try {
```

Comment:
Why cant we have one try catch exception here ?

Reply:
Each try catch will throw specific errors, that's why we add multiple try catches,

Reply:
since its the same exception , if you can merge both messages to make sense then do it otherwise its fine